modules/v2x/fusion/libs/common/base/comparison_traits.h

```diff
@@ -0,0 +1,47 @@
+/******************************************************************************
```

Comment:
This file is exactly the same as `modules/perception/base/box.h`. Why not just reuse it?